Example/AppDelegate.swift

```diff
@@ -28,6 +28,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             return MiniApp.MAOrientationLock
         }
     }
+
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        guard let components = URLComponents(url: url, resolvingAgainstBaseURL: true), let host = components.host else {
+            return false
+        }
+        print("Host: ", host)
+        return true
+    }
```

Comment:
This is already on master, I don't know why Github is displaying this as new code

Reply:
This branch is not updated yet buddy

Reply:
oh, I would have sworn it was ^^